.github/workflows/lint.yaml

```diff
@@ -50,3 +50,22 @@ jobs:
         VALIDATE_PYTHON_FLAKE8: false
         VALIDATE_PYTHON_MYPY: false
         VALIDATE_PYTHON_RUFF: false
+    - name: Super-Linter
+      uses: super-linter/super-linter/slim@b4515bd4ad9d0aa4681960e053916ab991bdbe96
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        # Set your fix mode variables to true
+        FIX_SHELL_SHFMT: true
```

Comment:
You don't need to run super linter again

Reply:
Removed the linter changes. Will add in a separate PR later.